WebRoot/js/ajax/dwt/xforms/XModelItem.js

```diff
@@ -440,7 +444,7 @@ XModelItem.prototype._normalizeAndValidate = function(value) {
     		}
     	}
     	if (!matched) {
-			throw this.getModel().getErrorMessage("stringMismatch", value);
+			throw this.getModel().getErrorMessage(stringMismatchErrorMessage, value);
```

Comment:
set custom message here.
```suggestion
			stringMismatchErrorMessage = this.getCustomErrorMessage() || "stringMismatch";
			throw this.getModel().getErrorMessage(stringMismatchErrorMessage, value);
```